src/javascript/binary/base/page.js

```diff
@@ -523,6 +523,27 @@ Header.prototype = {
             BinarySocket.send({"logout": "1"});
         });
     },
+    
+    validate_cookies: function(){
+        function is_valid_id(id){
+            return /^(MX|MF|VRTC|MLT|CR|VRTJ|JP)[0-9]+$/i.test(id);
```

Comment:
you can get the values from page.settings.get('valid_loginids'), it returns "MX|MF|VRTC|MLT|CR|FOG|VRTJ|JP'" so you don't have to hard code it